docs/source/cn/math_ops.py

```diff
@@ -41,3 +41,695 @@
 
     """,
 )
+
+reset_docstr(
+    oneflow.sub,
+    r"""sub(input: Tensor, other: Tensor) -> Tensor
+    
+    计算 `input` 和 `other` 的差，支持 element-wise、标量和广播形式的减法。
+    公式为：
+
+    .. math::
+        out = input - other
+    
+    示例：
+
+    .. code-block:: python
+
+        >>> import numpy as np
+        >>> import oneflow as flow
+        
+        # element-wise 减法
+        >>> input = flow.tensor(np.random.randn(2,3), dtype=flow.float32)
+        >>> other = flow.tensor(np.random.randn(2,3), dtype=flow.float32)
+        >>> out = flow.sub(input,other).numpy()
+        >>> out.shape
+        (2, 3)
+
+        # 标量减法
+        >>> input = 5
+        >>> other = flow.tensor(np.random.randn(2,3), dtype=flow.float32)
+        >>> out = flow.sub(input,other).numpy()
+        >>> out.shape
+        (2, 3)
+
+        # 广播减法
+        >>> input = flow.tensor(np.random.randn(1,1), dtype=flow.float32)
+        >>> other = flow.tensor(np.random.randn(2,3), dtype=flow.float32)
+        >>> out = flow.sub(input,other).numpy()
+        >>> out.shape
+        (2, 3)
+
+    """,
+)
+
+reset_docstr(
+    oneflow.abs,
+    r"""abs(input: Tensor) -> Tensor
+    
+    返回一个包含 `input` 中每个元素的绝对值的tensor:`y = |x|`。
+    
+    参数：
+        input (Tensor): 输入张量。
+
+    示例：
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        >>> import numpy as np
+
+        >>> x = flow.tensor(np.array([-1, 2, -3, 4]).astype(np.float32))
+        >>> flow.abs(x)
+        tensor([1., 2., 3., 4.], dtype=oneflow.float32)
+    
+    """,
+)
+
+reset_docstr(
+    oneflow.div,
+    r"""div(input: Tensor, other: Tensor) -> Tensor
+    
+    计算 `input` 除以 `other`，支持 element-wise、标量和广播形式的除法。
+    公式为：
+
+    .. math::
+        out = \frac{input}{other}
+    
+    参数：
+        input (Union[int, float, oneflow.Tensor]): input.
+        other (Union[int, float, oneflow.Tensor]): other.
```

Comment:
```suggestion
        - input (Union[int, float, oneflow.Tensor]): input.
        - other (Union[int, float, oneflow.Tensor]): other.
```
不加 `-` 的话，这两个就合并成同一行了（请自己严格检查编译结果）

Reply:
已解决